docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -197,6 +188,11 @@ A rule can have one of the following statuses:
 `pending`:: The rule has not yet run.  The rule was either just created, or enabled after being disabled.
 `unknown`:: A problem occurred when calculating the status. Most likely, something went wrong with the alerting code.
 
+Each alert also has one of the following statuses:
+
+`active`:: The conditions for the rule are met, and actions should be generated according to the notification settings.
+`recovered`:: The conditions for the rule are no longer met.
+
```

Comment:
I added this section since I couldn't find anywhere else we take about alert statuses. However, if those are indeed the only valid status options, I'm confused about when/if the "only on status change" option would ever fire. i.e. If the notify options pertain only to alerts are "active" and yet we're telling it to only repeat actions when the status changes, does that mean they only repeat when an alert has gone from "recovered" to "active"?

Reply:
Only on status change would only fire when alert goes from active to recovered or recovered to active. Some rules have multiple action groups, for example `error` and `warn` so both `error` and `warn` would be considered `active` but actions would fire if moving from `error` to `warn` or `warn` to `recovered` or `recovered` to `error`, for example.

Reply:
Thanks!  I have updated the text to (1) remove "while the alert is active" from the "Notify when" hover, and (2) updated the alert status docs to clarify that "active" and "recovered" are just two examples. If you think we ought to cover the other possibilities there as well, let me know.